test/framework/kubernetes.go

```diff
@@ -139,6 +143,21 @@ func IsPodRunning(pod *corev1.Pod) bool {
 	return pod.Status.Phase == corev1.PodRunning
 }
 
+// WaitForDeploymentRunning waits for a deployment to be running, with a specific number of pod
+func WaitForDeploymentRunning(namespace, dName string, podNb int, timeoutInMin int) error {
+	return WaitForOnKubernetes(namespace, fmt.Sprintf("Deployment %s running", dName), timeoutInMin,
```

Comment:
I think we should refactor the WaitForOnXXX since in theory, there is nothing to do with Kubernetes or Openshift, but a time factor that could be shared among clusters.